README.md

```diff
@@ -1,27 +1,31 @@
-# docker-weasyprint
+---
+title: WeasyPrint HTML to PDF Microservice
+description: A ready to use, OpenShift compatible, HTML to PDF microservice for your application. 
+---
+# Docker-WeasyPrint
```

Comment:
Easier to explain when we meet in person, but your main header should match the title description for best UX in the devhub. 

```md
---
title: WeasyPrint HTML to PDF Microservice
description: A ready to use, OpenShift compatible, HTML to PDF microservice for your application. 
---
# WeasyPrint HTML to PDF Microservice
...
```